Documentation/226 getting started and configuration (#227)
exasol/pyexasol: doc/user_guide/configuration/local_configuration.rst

> Configuration File ``pyexasol.ini``

Reviewer: ```suggestion
Configuration File ``.pyexasol.ini``
```

===================================

> Local configuration is a popular feature among data scientists and analysts working on laptops.
> It allows users to store personal Exasol credentials and connection options in a local file, which is separated from the code.

Reviewer: ```suggestion
Using a local configuration file is a popular feature among data scientists and analysts working on laptops.
It allows users to store personal Exasol credentials and connection options separate from the code.
```
